src/utils/Utils.ts

```diff
@@ -10,7 +10,7 @@ export class Utils {
     return records
       .filter((record: any) => {
         // Filter by testStatus
-        return record.testStatus === "submitted";
+        return record.testStatus === "submitted" || record.testStatus === "cancelled";
```

Comment:
I don't _think_ there could be any other test statuses. If we do have other statuses maybe we have bigger problems, although you could argue it's safer this way. 

So yeah I think we could just delete that filter altogether

Reply:
I thought about this, and I just think this is safer and doesn't really cause harm either way.

Reply:
fair enough